core/catkin_ws/src/custom_msgs/action/AcousticsWrapper.action

```diff
@@ -3,6 +3,12 @@
 # Target frequency
 uint32 tar_f
 
+# Use Saleae or Generated Data
+bool gen_data
+
+# Location of the pinger (only if using generated data)
+geometry_msgs/Point location
+
```

Comment:
Remove these, you should not be modifying the AcousticsWrapper action, since this is the external interface with which task planning communicates with acoustics, so they should not be confused by this. Instead, create a new action, that will be used with the wrapper if using generated data.